packages/frontend/web-ui/src/auth/hooks/useLoginForm.spec.ts

```diff
@@ -0,0 +1,313 @@
+import { describe, expect, jest, it, beforeAll, afterAll } from '@jest/globals';
+
+jest.mock('../../common/helpers/validateEmail');
+jest.mock('../../common/helpers/validatePassword');
+jest.mock('../helpers/isFullfilledPayloadAction');
+jest.mock('react-redux', () => ({
+  ...(jest.requireActual('react-redux') as Record<string, unknown>),
+  useDispatch: jest.fn(),
+}));
+
```

Comment:
Im missing a `createAuthByCredentials` mock

Reply:
Is that necessary? I mean, you mock useDispatch and the PayloadAction.

Reply:
Yes, it is. We should not rely on other modules logic if possible

Reply:
Done!